### PR TITLE
Feature/create activity

### PR DESCRIPTION
**Pull request by MihaiDruta**
[BE] Add Job
https://trello.com/c/ejzp0pzp/10-be-add-job

Added functions for creating a new activity.

### CODE REVIEW

**Review by nspc97 — APPROVED** [review]

File: src/DTO/ActivityDTO.php [hunk 1]
```diff
@@ -0,0 +1,108 @@
...
+     * @var integer
+     * @Serializer\Type("integer")
+     */
+    public $id;
+
+    /**
+     * @var string
+     * @Serializer\Type("string")
+     * @Assert\NotBlank(
+     *     message = "Activity name cannot be blank!")
```
Comment (SpartakusMd) [inline src/DTO/ActivityDTO.php]: If you start to use new lines in an annotation, make sure to put the closing parenthesis `)` on a new line.

File: src/DTO/TypeDTO.php [hunk 2]
```diff
@@ -0,0 +1,51 @@
+<?php
+
+namespace App\DTO;
+
+use DateTime;
+use JMS\Serializer\Annotation as Serializer;
+
+class TypeDTO
```
Comment (SpartakusMd) [inline src/DTO/TypeDTO.php]: As we have disscussed, please rename the DTO and the entity. `Type` is too broad. Maybe `ActivityType` or something else.

File: src/DTO/TechnologyDTO.php [hunk 3]
```diff
@@ -0,0 +1,50 @@
...
+     */
+    public $updatedAt;
+
+    public function __construct()
+    {
+        $this->createdAt = new DateTime();
+        $this->updatedAt = new DateTime();
+    }
+
+    public function getId(): ?int
```
Comment (SpartakusMd) [inline src/DTO/TechnologyDTO.php]: Why you have this method if the field is public?

File: src/DTO/TypeDTO.php [hunk 4]
```diff
@@ -0,0 +1,51 @@
...
+     */
+    public $updatedAt;
+
+    public function __construct()
+    {
+        $this->createdAt = new DateTime();
+        $this->updatedAt = new DateTime();
+    }
+
+    public function getId(): ?int
```
Comment (SpartakusMd) [inline src/DTO/TypeDTO.php]: Why you have this method if the field is public?

**Review by SpartakusMd — CHANGES_REQUESTED** [review]

**Review by SpartakusMd — APPROVED** [review]

**Pull request merged**